addons/refuel/functions/fnc_takeNozzle.sqf

```diff
@@ -55,10 +55,13 @@ if (isNull _nozzle) then { // func is called on fuel truck
             _newNozzle attachTo [_unit, [-0.02,0.05,-0.12], "righthandmiddle1"];
             _unit setVariable [QGVAR(nozzle), _newNozzle, true];
 
-            private _rope = ropeCreate [_target, _endPosOffset, _newNozzle, [0, -0.20, 0.12], REFUEL_HOSE_LENGTH];
+            if (_target isKindOf "AllVehicles") then {
+                // Currently ropeCreate requires its first parameter to be a real vehicle
+                private _rope = ropeCreate [_target, _endPosOffset, _newNozzle, [0, -0.20, 0.12], REFUEL_HOSE_LENGTH];
+                _newNozzle setVariable [QGVAR(rope), _rope, true];
+            };
```

Comment:
If we cannot create a rope, doesn't that mean something is wrong and we cannot properly let a user take the fuel nozzle?